arch/arm/boot/dts/overlays/README

```diff
@@ -3102,6 +3102,8 @@ Params: gpiopin                 Input GPIO (default "18")
                                 off)
         capture_clear           Generate clear events on the trailing edge
                                 (default off)
+        pull                    Specify whether the pin is pulled up (2), down
```

Comment:
You don't need the magic numbers - the values `up`, `down` and `off` are also accepted (and preferred) the parameter mechanism converts them to the appropriate integers.

Reply:
Is this a suggested documentation fix, or would it require an actual change to the .dts commit? Struggling to find documentation of this other than the existing dtbo readme sections, which have precedents for both ways.

Reply:
It's just a documentation change. The overlay handling code supports several string-to-integer conversions:
* `off`, `no`, `n`, `false` -> 0
* `on`, `yes`, `y`, `true` , `down` -> 1
* `up` -> 2

Reply:
Thanks, updated documentation (and tested it both ways for good measure).